openapi/azure.sqldb/Module.md

```diff
@@ -0,0 +1,9 @@
+## Overview
+This is a generated connector from [Azure SQL DB API v1.0](https://docs.microsoft.com/en-us/azure/azure-sql/database/sql-database-paas-overview) OpenAPI specification.
```

Comment:
Please verify whether the correct API version is used in the Package.md, openapi.yml, client.bal and here.

Reply:
Did the suggested changes.